pyproject.toml

```diff
@@ -28,43 +30,43 @@ Issues = "https://github.com/applied-ml-bde/bde/issues"
 [tool.setuptools_scm]
 version_file = "bde/_version.py"
 
-[tool.poetry]
```

Comment:
I have some setup difficulties with pixi. Please provide instructions in the Readme!

Reply:
I updated the [development setup](https://github.com/applied-ml-bde/bde/tree/11-implement-mcmc-stage?tab=readme-ov-file#development-setup) section in the readme file.
Do you think this is clear enoguh, or do you recommend I change it up?

Reply:
It's ok. @SteffenJw should be able to reproduce without problems. Is this the case?

Reply:
> It's ok. @SteffenJw should be able to reproduce without problems. Is this the case?

yes, it works. I had redo it on a different machine because bc my pc needs new hardware. 
I just used the install commands provided in the link. 
Just dont do the intuitive pip install pixi. Maybe there should be a warning . As far as I understand, that installs something completely different.

Reply:
(i installed it on windows if that matters)